services/crudService.js

```diff
@@ -0,0 +1,57 @@
+const errorService = require('./errorService');
+
+const findAll = (model, res, isDataSend = true) => {
+    return model
+        .find({})
+        .then(data => isDataSend ? res.status(200).send(data) : data)
+        .catch(err => errorService(res, err.message));
+};
+
+const findById = (model, res, id, isDataSend = true) => {
+    return model
+        .findById(id)
+        .then(data => isDataSend ? res.status(200).send(data) : data)
+        .catch(err => errorService(res, err.message));
+};
+
+const save = (model, res, body, isDataSend = true) => {
+    return model(body)
+        .save()
+        .then(data => {
+            const msg = `${ model.modelName } is saved`;
+            return isDataSend ? res.status(200).send({ msg }) : data;
+        })
+        .catch(err => errorService(res, err.message));
+};
+
+const updateById = (model, res, body, isDataSend = true) => {
+    return model
+        .findOneAndUpdate(
+            { _id: body._id },
```

Comment:
ID should be taken from URL params insteab of body.
_id in body should be ignored.